react/__mocks__/vtex.render-runtime.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 
-export function withRuntimeContext(Comp) {
-  return Comp
-}
+export const withRuntimeContext = Comp => props => (
```

Comment:
Note: In case of you agreeing with my suggestions, this would not be necessary anymore.

Reply:
This mock is been used for another components as well.

Reply:
Oh, okay 😄